pegasus/sites.v3/code.org/public/amazon-future-engineer.md.erb

```diff
@@ -62,3 +72,11 @@ Take your teaching to the next level with a CSTA+ membership sponsored by Amazon
 # Sign-up today to get started!
 
 <%= view :donor_teacher_banner %>
+
+[/show-if-eligible]
+
+[show-if-not-eligible]
+
+Sorry, this program is only available to select schools. If you have any questions, please contact us [here](https://support.code.org/hc/en-us/requests/new).
```

Comment:
Should we add instruction for users to specify their schools if they haven't done that yet.

Reply:
Unfortunately, there is currently no interface for allowing users to update their school info.

Reply:
Bryan elaborated in person: We don't have a user-friendly way for users to do this.  There is a `?forceSchoolInfoInterstitial=true` queryparam we've used for testing, but I'm not sure I trust it enough to include a link to it here.

Reply:
Do we know if majority of teachers from those 700 schools had set their school info (and set them correctly I hope)? I'm worry because teachers could skip school info section when they signed up. And now they will see "Unavailable" message even if they are from eligible schools.

Reply:
That's a good consideration, Ha. The initial estimation for the program that Alice and Ben performed took into account just the teachers that we knew were at these 700 schools (~800 users), so we're in a good place for the number of teachers we want to target. If any teachers neglected to add their school info but are actually at one of these schools, they can get in touch with us via the support link.